### DISCUSSION TITLE
Doks v0.3.5 is here!

### DISCUSSION THREAD

**Discussion by h-enk** [thuliteio/doks#515] [General]
Doks now is fully prepared for multilingual and i18n support, fully prepared for versioned documentation support, and now you can also (next to using the image shortcodes) add an image using markdown (portability!). [Check the blog post](https://getdoks.org/blog/doks-v0.3/) or [get the new release](https://github.com/h-enk/doks/releases/tag/v0.3.5).

Up next is our v0.4.0 release, adding subnavigation to menus (main + section), automatically generating menus (main + section), and updating blog pages (list + single). [Check the v0.4.0 milestone](https://github.com/h-enk/doks/milestone/3).

Help support the team contributing to Doks. [Visit our Open Collective page](https://opencollective.com/doks).

**Comment by janprill** [comment]
Thanks a million for this great theme! Your effort is highly appreciated!

**Reply by h-enk** [reply]
That super great to hear!
